lib/util.js

```diff
@@ -352,6 +352,10 @@ exports.stripHTML = html => html
   .replace(/[\n\r]/g, ' ')
   .replace(/\s*<\s*br\s*\/?\s*>\s*/gi, '\n')
   .replace(/<\s*\/\s*p\s*>\s*<\s*p[^>]*>/gi, '\n')
+  .replace(/<a\s+(?:[^>]*?\s+)?href=(?:["'])\/redirect.*?q=(.*?)(?:[&'"]).*?<\/a>/gi,
+    (_, p1) => decodeURIComponent(p1) || p1)
+  .replace(/<a\s+(?:[^>]*?\s+)?href=(?:["'])((?:https?|\/).*?)(?:['"]).*?<\/a>/gi,
+    (_, p1) => url.resolve('https://youtube.com/', p1))
```

Comment:
does this need `decodeURIComponent` too?

Reply:
url.resolve already decodes stuff